content/develop/ai/search-and-query/vectors.md

```diff
@@ -129,6 +129,55 @@ FT.CREATE documents
 
 In the example above, an index named `documents` is created over hashes with the key prefix `docs:` and an `HNSW` vector field named `doc_embedding` with five index attributes: `TYPE`, `DIM`, `DISTANCE_METRIC`, `M`, and `EF_CONSTRUCTION`.
 
+### SVS-VAMANA index
+
+Scalable Vector Search (SVS) is an Intel project in which a new vector search library, VAMANA graph index, was created. SVS-VAMANA supports highly accurate compressed vector indexes. You can read more about the project [here](https://intel.github.io/ScalableVectorSearch/intro.html). Support for `SVS-VAMANA` indexing was added in Redis 8.2.
+
+Choose the `SVS-VAMANA` index type when you need vector search
+
+- on billions of high-dimensional vectors,
+- at high accuracy and state-of-the-art speed,
+- using less memory than alternatives.
```

Comment:
We should include a more comprehensive explanation that highlights the differences between HNSW and VAMANA, as well as when to choose each.
I suggest starting with a short description of the vanilla SVS-VAMANA algorithm, then emphasizing that its main advantage is the ability to use it with the `COMPRESSION` parameter to reduce memory usage.
That’s mentioned here, but I think it could be clearer.
@alonre24 WDYT? Also, can we ask them for a simple guideline table that outlines when to choose each compression type, depending on the data characteristics?